source/features/preview-markdown-result.tsx

```diff
@@ -0,0 +1,31 @@
+import './preview-markdown-result.css';
+import React from 'dom-chef';
+import delegate from 'delegate-it';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function togglePreviewResult({delegateTarget: target}: delegate.Event<MouseEvent, HTMLButtonElement>): void {
+	document.body.classList.toggle('rgh-preview-markdown-result');
```

Comment:
Everything looks good. I’m just concerned that the state of this might lose sync with the buttons. I suggest adding a `value=true/false` attribute on the buttons so you can use a safer 
```suggestion
	document.body.classList.toggle('rgh-preview-markdown-result', Boolean(target.value));
```

Reply:
I ended up using `'enabled'`/`'disabled'` because `Boolean('false')` is still `true`.